cypress/e2e/tests/footer.cy.js

```diff
@@ -0,0 +1,220 @@
+context('Footer Get Started', () => {
+  beforeEach(() => {
+    cy.visit('/')
+    cy.get('div')
+      .contains('Get Started')
+      .siblings('div')
+      .as('getStarted')
+  })
+
+describe('Voice', () => {
+  it('Should verify the external link exists', () => {
+    cy.get('@getStarted')
+      .contains('Voice')
+      .should('have.attr', 'href', 'https://www.bandwidth.com/voice/')
+    })
+  })
+
+describe('Messaging', () => {
+  it('Should verify the external link exists', () => {
+    cy.get('@getStarted')
+      .contains('Messaging')
+      .should('have.attr', 'href', 'https://www.bandwidth.com/messaging/')
```

Comment:
The base URL  `https://www.bandwidth.com` should be extracted to configuration or run time variable. We can't re-use these Tests without it.

Reply:
Here is the example I was talking about; this is an external link to the messaging page on Bandwidth's main site so this won't ever change.

Reply:
Here is the example I was talking about; this is an external link to the messaging page on Bandwidth's main site so this won't ever change.